lib/avro-patches/schema_validator/schema.rb

```diff
@@ -0,0 +1,9 @@
+Avro::Schema.class_eval do
```

Comment:
The only clobbering was was intentional, as I was attempting to layer these changes.

There was actually relatively little overlap between the various patches.

I included all of the tests in an attempt to ensure that we weren't losing anything, and my confidence in the tests is good.